bindings/java/mongocrypt/src/main/java/com/mongodb/crypt/capi/MongoCrypts.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.mongodb.crypt.capi;
+
+/**
+ * The entry point to the MongoCrypt library.
+ */
+public class MongoCrypts {
+
+    /**
+     * Create a {@code MongoCrypt} instance.
+     *
+     * <p>
+     * Make sure that JNA is able to find the shared library, most likely by setting the jna.library.path system property
```

Comment:
Consider adding a libraryPath option, similar to the one in the embedded [MongoClients](https://github.com/mongodb/mongo-java-driver/blob/master/driver-embedded/src/main/com/mongodb/embedded/client/MongoClients.java#L48-L49).  Then you can set it [explicitly](https://github.com/mongodb/mongo/blob/master/src/mongo/embedded/mongo_embedded/java/src/com/mongodb/embedded/capi/MongoEmbeddedCAPI.java#L80-L82) rather than just rely on system properties.

Also, I would recommend wrapping a call to `new CAPI();` to test if there are any JNA issues. Similar to [here](https://github.com/mongodb/mongo/blob/master/src/mongo/embedded/mongo_embedded/java/src/com/mongodb/embedded/capi/MongoEmbeddedCAPI.java#L83-L92);

Reply:
I'd like to defer this until we're convinced that we can't always load from the shared library from inside the jar.

Reply:
ACK